src/actions/tags.js

```diff
@@ -51,7 +51,7 @@ export function createTagInCommunity (params, slug) {
   return {
     type: CREATE_TAG_IN_COMMUNITY,
     payload: {api: true, params, path: `/noo/community/${slug}/tag/`, method: 'POST'},
-    meta: {slug, optimistic: true}
+    meta: {slug, tag: params, optimistic: true}
```

Comment:
calling it params seems pretty standard, see eg updatePost and createPost. Am I missing a difference between those and this? Or are they out of date now and should also be clarified?